src/crewmate.cpp

```diff
@@ -0,0 +1,20 @@
+#include "crewmate.hpp"
+#include "world.hpp"
+
+#include <iostream>
+
+Crewmate::Crewmate(const World &w, std::string_view name_, Gender gend)
+	: name(name_), gender(gend), world(w)
+{
+	birthTimestamp = world.GetTime();
+}
+void Crewmate::Update(sf::Time deltaTime)
+{
+	// Quick check if adult or not
+	if (!adult && (world.GetTime().Year() - birthTimestamp.Year()) >= AdultYear)
```

Comment:
I think the "isAdult()" function should compute that instead of saving it in a bool